citi-onboarding/Maracatec
# Feature/style adjustments

What i did 
-----------
I created the scroll links on navbar and footer 

How to Test 
------------

On Terminal:

1. ` git pull` do it on main
2. ` git checkout feature/style-adjustments` 
3. ` git pull`
4. ` yarn start`

To checkout this, there is no need to put the docker-compose up. Just see if the links are going in to the right places
